core/lib/env_config/src/da_client.rs

```diff
@@ -110,8 +120,8 @@ mod tests {
             DA_API_NODE_URL="localhost:12345"
             DA_BRIDGE_API_URL="localhost:54321"
             DA_APP_ID="1"
-            DA_TIMEOUT="2"
-            DA_MAX_RETRIES="3"
+            DA_GAS_RELAY_MODE="false"
+            DA_GAS_RELAY_API_URL="localhost:23456"
```

Comment:
this seems like it won't work, please try running `cargo test` in this directory to make sure it passes